api/bazel/BUILD

```diff
@@ -1,4 +1,7 @@
 load("@io_bazel_rules_go//proto:compiler.bzl", "go_proto_compiler")
+load("@envoy//bazel:utils.bzl", "write_json")
+load("@envoy//tools/base:envoy_python.bzl", "py_data")
```

Comment:
Why not put both of these in `//bazel:utils.bzl`? This is kind of core.

Reply:
i put the python stuff here as that was where the other python utils are, but i can move